docs/en/engines/table-engines/integrations/iceberg.md

```diff
@@ -6,28 +6,34 @@ sidebar_label: Iceberg
 
 # Iceberg Table Engine
 
-This engine provides a read-only integration with existing Apache [Iceberg](https://iceberg.apache.org/) tables in Amazon S3.
+This engine provides a read-only integration with existing Apache [Iceberg](https://iceberg.apache.org/) tables in Amazon S3, Azure and locally stored tables.
```

Comment:
BTW it should be now very quick as well to add local/azure storage for DeltaLake and Hudi. I think it makes sense and should take like a few minutes to add.

Reply:
It is a matter of testing. It shouldn't be difficult but can require several hours